.github/workflows/test.yml

```diff
@@ -240,6 +248,13 @@ jobs:
           VIKUNJA_AUTH_LDAP_BINDPASSWORD: password
           VIKUNJA_AUTH_LDAP_USERFILTER: "(&(objectclass=inetorgperson)(uid=%s))"
           VIKUNJA_SERVICE_PUBLICURL: http://127.0.0.1:3456
+          VIKUNJA_FILES_TYPE: s3
```

Comment:
Can this be local by default? Then in the s3 integration test, set it to `s3` and maybe fail when no endpoint is configured.

Right now it would be possible to run the s3 integration tests with local storage, which does not really make sense.

Reply:
<img width="976" height="614" alt="image" src="https://github.com/user-attachments/assets/9aeb7b23-e7c3-4505-8186-033b91c8164f" />